solr/solr-ref-guide/src/collection-management.adoc

```diff
@@ -22,7 +22,42 @@ A collection is a single logical index that uses a single Solr configuration fil
 [[create]]
 == CREATE: Create a Collection
 
-`/admin/collections?action=CREATE&name=_name_`
+The CREATE action is used to create new collection of data.
+
+[.dynamic-tabs]
+--
+[example.tab-pane#v1createcollection]
+====
+[.tab-label]*V1 API*
+
+With the v1 API, the `create` command must be capitalized as `CREATE`:
+
+[source,bash]
+----
+http://localhost:8983/solr/admin/collections?action=CREATE&name=newCollection
+
+----
+====
+
+[example.tab-pane#v2createcollection]
+====
+[.tab-label]*V2 API*
+
+With the v2 API, the `create` command is provided as part of the JSON data that contains the required parameters:
+
+[source,bash]
+----
+curl -X POST http://localhost:8983/api/collections?omitHeader=true -H 'Content-Type: application/json' -d'
+  {
+    "create": {
+      "name": "newCollection",
+      "numShards": 1
```

Comment:
Thanks for reviewing!   So, I just tried the v1 api call, and it turns out NOW you need to have numShards, otherwise you get `"numShards is a required param (when using CompositeId router).`, so updating the v1 api example.